modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module CovidResearch
+  module Volunteer
+    class FormCryptoService
+      extend AttrEncrypted
+      attr_encrypted :submission, key: Settings.db_encryption_key
+
+      def decrypt_redis_format(redis_format)
```

Comment:
A comment and/or yardoc here would be helpful. Perhaps something like:
```suggestion
      # @param redis_format [Hash] a Hash with encrypted and Base64-encoded submission and iv
      # @return [String] decrypted form data
      def decrypt_redis_format(redis_format)
```